### ISSUE TITLE
Add in logs messages information about jobs

### ISSUE THREAD

**Issue by elhoim** [MISP/MISP#813]
The log information does not contain the job ID. For example I was trying to correlate past failed jobs, and get more context about them from the logs, but I could not since the log messages for pulls do not contain job IDs.
I suppose it is the same for all types of jobs (push & generate exports).

**Comment by iglocska** [comment]
Yeah, we should definitely update the logging to include a reference to the jobs in the future.

**Comment by elhoim** [comment]
Triage note (checked against `2.5` @ `843b674450`): still not implemented — `Server::pull()` and `Server::push()` both receive a `$jobId` and use it for progress reporting, then drop it when writing the summary log entry ([`app/Model/Server.php:723`](https://github.com/MISP/MISP/blob/843b674450/app/Model/Server.php#L723) and [`:1475`](https://github.com/MISP/MISP/blob/843b674450/app/Model/Server.php#L1475)).

Opened **MISP/MISP#10980**, which appends ` (job #N)` to both log titles when the operation runs as a background job (and, for #911, splits the pull counters). Lint-clean; no live instance was available, so an actual sync has not been run.
